ui/mirage/handlers/activity.js

```diff
@@ -2,6 +2,7 @@ export default function (server) {
   // 1.10 API response
   server.get('/sys/internal/counters/activity', function () {
     return {
+      request_id: '26be5ab9-dcac-9237-ec12-269a8ca647d5',
```

Comment:
Claire, heads up that I added this to the mirage handler. Needed for making my queryRecord, it needed an ID.

Reply:
I added to my open PR too, so should be good!